### PR TITLE
[stable/fluent-bit] Cater for full set of Elastic Output Plugin parameters (Auth & TLS)

### PR DESCRIPTION
**Pull request by MnrGreg**
Caters for the full set of parameters for the Fluent-Bit Elastic Output Plugin. Useful when logging to an external Elastic platform.

Parameters include:
- X-Pack Authentication
- TLS

Mentioning maintainers
@kfox1111
@edsiper

### CODE REVIEW

File: stable/fluent-bit/values.yaml [hunk 1]
```diff
@@ -16,6 +16,21 @@ backend:
...
+    # Elastic Index Name
+    index: kubernetes_cluster
+    type: flb_type
+    logstash_prefix: kubernetes_cluster
+    # Optional username credential for Elastic X-Pack access
+    http_user: ""
+    # Password for user defined in HTTP_User
+    http_passwd: ""
+    tls: off
+    tls_verify: off
```
Comment (timstoop) [inline stable/fluent-bit/values.yaml]: Please add some documentation for these options, preferably make a start with the documentation in the README, as it's currently not very stable-worthy.

File: stable/fluent-bit/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: fluent-bit
-version: 0.2.13
+version: 0.2.14
```
Comment (timstoop) [inline stable/fluent-bit/Chart.yaml]: This adds features, please bump the minor version.

**Review by timstoop — CHANGES_REQUESTED** [review]

File: stable/fluent-bit/values.yaml (hunk 1 above)
Comment (MnrGreg) [reply stable/fluent-bit/values.yaml]: README updated

File: stable/fluent-bit/Chart.yaml (hunk 2 above)
Comment (MnrGreg) [reply stable/fluent-bit/Chart.yaml]: Changed version to 0.2.13.1. Is that inline with what you're suggesting?

Comment (timstoop) [reply stable/fluent-bit/Chart.yaml]: No, I was suggesting bumping it to 0.3.0.

Comment (MnrGreg) [reply stable/fluent-bit/Chart.yaml]: Bumped. Thanks for clarifying.

**Comment by MnrGreg** [comment]
@unguiculus is anything else needed from me to merge this PR?

**Comment by timstoop** [comment]
Merge from master again please and fix the Chart.yaml. There has been another merge to this Chart, apparantly.

**Comment by MnrGreg** [comment]
@timstoop @unguiculus merged with master and bumped the chart version again.

thanks

**Comment by MnrGreg** [comment]
/assign @unguiculus

Danke schön Reinhard

**Comment by unguiculus** [comment]
@MnrGreg Since you changed the RBAC default, would you mind updating RBAC to our current best practices as well?

https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/rbac.md

**Comment by MnrGreg** [comment]
@timstoop @unguiculus update RBAC and ServiceAccount to standards. Merged with master again and bumped the chart version

**Comment by unguiculus** [comment]
Please bump the minor version. Also, the cluster role binding needs to be updated for the service account.

**Comment by MnrGreg** [comment]
@timstoop @unguiculus bumped minor version and set ClusterRoleBinding to use serviceAccountName

**Pull request merged**